### PR TITLE
fix: preload all vscode-neovim lua modules

### LINKED ISSUE #2200: bug: module 'vscode.internal' not found
### Check the following:

- [ ] I have checked the behavior after setting "vscode-neovim.neovimClean" in VSCode settings.
- [X] I have read the vscode-neovim docs.
- [X] I have searched existing issues.

### Neovim version (nvim -v)

0.9.4

### Operating system/version

Ubuntu 22.04

### Describe the bug

extension does not work when starting the vs code and displays this error message

```
Error syncing layout: [Error: nvim_execute_lua: Error executing lua: [string "<nvim>"]:1: module 'vscode.internal' not found:
	no field package.preload['vscode.internal']
cache_loader: module vscode.internal not found
cache_loader_lib: module vscode.internal not found
	no file './vscode/internal.lua'
	no file '/__w/neovim/neovim/.deps/usr/share/luajit-2.1/vscode/internal.lua'
	no file '/usr/local/share/lua/5.1/vscode/internal.lua'
	no file '/usr/local/share/lua/5.1/vscode/internal/init.lua'
	no file '/__w/neovim/neovim/.deps/usr/share/lua/5.1/vscode/internal.lua'
	no file '/__w/neovim/neovim/.deps/usr/share/lua/5.1/vscode/internal/init.lua'
	no file './vscode/internal.so'
	no file '/usr/local/lib/lua/5.1/vscode/internal.so'
	no file '/__w/neovim/neovim/.deps/usr/lib/lua/5.1/vscode/internal.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file './vscode.so'
	no file '/usr/local/lib/lua/5.1/vscode.so'
	no file '/__w/neovim/neovim/.deps/usr/lib/lua/5.1/vscode.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
	[C]: in function 'require'
	[string "<nvim>"]:1: in main chunk
	[C]: in function 'get_config'
	...ov.vscode-neovim-1.18.7/runtime/lua/vscode/highlight.lua:33: in function 'setup_globals'
	...ov.vscode-neovim-1.18.7/runtime/lua/vscode/highlight.lua:165: in function <...ov.vscode-neovim-1.18.7/runtime/lua/vscode/highlight.lua:162>
	[C]: in function 'nvim_exec2'
	vim/_editor.lua: in function 'cmd'
	/home/jaydev/.config/nvim/lua/jaydev/dependencies.lua:25: in function 'config'
	...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:366: in function <...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:364>
	[C]: in function 'xpcall'
	.../.local/share/nvim/lazy/lazy.nvim/lua/lazy/core/util.lua:113: in function 'try'
	...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:381: in function 'config'
	...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:348: in function '_load'
	...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:191: in function 'load'
	...local/share/nvim/lazy/lazy.nvim/lua/lazy/core/loader.lua:123: in function 'startup'
	...aydev/.local/share/nvim/lazy/lazy.nvim/lua/lazy/init.lua:112: in function 'setup'
	/home/jaydev/.config/nvim/lua/jaydev/dependencies.lua:2: in main chunk
	[C]: in function 'require'
	/home/jaydev/.config/nvim/lua/jaydev/init.lua:4: in main chunk
	[C]: in function 'require'
	/home/jaydev/.config/nvim/init.lua:1: in main chunk
	at /home/jaydev/.vscode/extensions/asvetliakov.vscode-neovim-1.18.7/dist/extension.js:2:129307
	at process.processTicksAndRejections (node:internal/process/task_queues:95:5)]
```
I have to disable and enable the extension to make it work. happens everytime i start the vs code.

### Steps To Reproduce

Just start the VS code this error should be visible

### Expected Behavior

There should be no error and nvim should work.

### PR DESCRIPTION
**Pull request by xiyaowong**
fix #2200

### CODE REVIEW

File: runtime/lua/vscode.lua [hunk 1]
```diff
@@ -1,3 +1,13 @@
+-- Preload all vscode-neovim modules
+-- Some plugin managers customize the loader, which can cause issues
```
Comment (justinmk) [inline runtime/lua/vscode.lua:2]: what plugin manager? if they fuck with the loader and break things, that's their fault.

Comment (xiyaowong) [reply runtime/lua/vscode.lua:2]: From the error message in the issue, lazy.nvim is being used, and lazy.nvim indeed has a custom loader, but I haven't looked into it deeply.

Comment (theol0403) [reply runtime/lua/vscode.lua:2]: lazy.nvim has worked for me and the hundreds of other people using lazy.nvim with vscode neovim

Comment (xiyaowong) [reply runtime/lua/vscode.lua:2]: Yeah, I'm also using lazy.nvim. But that doesn't mean it won't go wrong.

**Comment by xiyaowong** [comment]
I'm not sure if this will prevent related issues from occurring, but I agree that this isn't a fundamental solution.

**Pull request closed without merging**